packages/cfworkers/src/mod.test.ts

```diff
@@ -0,0 +1,79 @@
+import type {
+  KVNamespace,
+  Queue,
+} from "@cloudflare/workers-types/experimental";
+import { assertEquals } from "@std/assert";
+import { delay } from "es-toolkit";
+import { WorkersKvStore, WorkersMessageQueue } from "./mod.ts";
+
+Deno.test({
```

Comment:
In order to make this test runnable on Node.js and Bun, we should use [`node:test`](https://nodejs.org/api/test.html) and [`node:assert/strict`](https://nodejs.org/api/assert.html).